src/core/histogram.cc

```diff
@@ -96,6 +104,65 @@ histogram::print(FILE *fp) const {
                                  CLASSIC);  // Format CLASSIC/CSV supported.
 }
 
+seastar::metrics::histogram
+histogram::get_seastar_metrics_histogram() const {
+  // logarithmic histogram configuration. this will range from 10 microseconds
```

Comment:
should we rename then to `histogram::logform_seastar_histogram()` if we are using the log form - which i very much like.